app/src/main/java/org/gdg/frisbee/android/api/PlusPersonDownloader.java

```diff
@@ -51,7 +51,12 @@ public Response intercept(Chain chain) throws IOException {
     }
 
     @Nullable
-    public Person getPersonSync(final String gplusId) {
+    public static Person getPersonSync(final String gplusId) {
```

Comment:
Plus Client is not a lightweight object as far as I understand. It requires http transport which requires okhttp and gson. 
I moved all PlusClient instances to one instance in App class.